Meshtastic/Views/Nodes/TraceRouteLog.swift

```diff
@@ -225,6 +225,10 @@ struct TraceRouteLog: View {
 					VStack {
 						let nodeColor = UIColor(hex: UInt32(truncatingIfNeeded: hops[i].num))
 						CircleText(text: String(hops[i].num.toHex().suffix(4)), color: Color(nodeColor), circleSize: idiom == .phone ? 70 : 125)
+							Text(String(hops[i].name ?? "Unknown"))
```

Comment:
`Unknown` is not localized:

```suggestion
							Text(String(hops[i].name ?? "Unknown"))
```